cabal.project

```diff
@@ -48,6 +48,36 @@ packages:
   doc/cookbook/using-free-client
   -- doc/cookbook/open-id-connect
 
-tests: True
+tests: False
 optimization: False
 -- reorder-goals: True
+
+-- https://github.com/vincenthz/hs-memory/pull/93
+source-repository-package
+    type: git
+    location: https://github.com/parsonsmatt/hs-memory
+    tag: 296b79424854eae293f6ba09b5308a0bf4dfd6d5
```

Comment:
This can be dropped now: https://hackage.haskell.org/package/memory-0.18.0/changelog